### PR TITLE
Add section on vulnerability scanning

### PR DESCRIPTION
**Pull request by stephen-turner**
Do not merge until this functionality reaches Desktop Stable

### CODE REVIEW

**Comment by stephen-turner** [comment]
@mkvitnitsky Please review text and placement. Thank you.

**Comment by stephen-turner** [comment]
From Marina: Stephen, the tutorial looks good...  Usha should review it also, but otherwise, I think we are good to go.

File: docs/tutorial/image-building-best-practices/index.md [hunk 1]
```diff
@@ -1,3 +1,45 @@
+## Security Scanning
+
+When you have built an image, it is good practice to scan it for security vulnerabilities, using the `docker scan` command.
```
Comment (usha-mandya) [inline docs/tutorial/image-building-best-practices/index.md:3]:
```suggestion
When you have built an image, it is good practice to scan it for security vulnerabilities using the `docker scan` command.
```

File: docs/tutorial/image-building-best-practices/index.md [hunk 2]
```diff
@@ -1,3 +1,45 @@
...
+Docker has partnered with [Snyk](http://snyk.io) to provide the vulnerability scanning service.
+
+For example, to scan the `getting-started` image you created earlier in the tutorial, you can just type
+```bash
```
Comment (usha-mandya) [inline docs/tutorial/image-building-best-practices/index.md:7]: nit: MD expects a blank line before the code block.

**Comment by usha-mandya** [comment]
LGTM. Just noted some minor observations.

**Comment by stephen-turner** [comment]
Thanks, @usha-mandya.

**Review by stephen-turner — COMMENTED** [review]
Marina and Usha have already reviewed, so merging in anticipation of today's release.

**Pull request merged**